### PR TITLE
HDS-2188 Update getting started as a designer guidelines

### PR DESCRIPTION
**Pull request by nooranen**
## Description

Figma era updates for the documentation site.

## Add to changelog
- [Changed] Updated Getting started-section for designers from Sketch/Abstract guidelines to Figma guidelines

### CODE REVIEW

**Comment by nooranen** [comment]
Should have been patient - need to change the Ui kit mentions to Design kit :)

**Review by arpargo — APPROVED** [review]
LGTM

File: site/src/docs/getting-started/designer.mdx [hunk 1]
```diff
@@ -30,58 +30,42 @@ Always keep these core principles in mind when making design decisions:
...
-- <ExternalLink href="https://share.goabstract.com/e6aa2ebd-1a49-4cb1-90ef-a44a6486c28b">Abstract Collection - Example Custom Components</ExternalLink>
-
-### Customising component symbols
-The component symbols use _smart layout_ for easy resizing. You can also configure the content and styling of symbol parts from the _Overrides_ section of symbol properties.
+For designers, a good place to start is to browse through <ExternalLink href="https://www.figma.com/file/Kxwg3R0zNRHj55nQqFu6VS/HDS-UI-Kit-v3?type=design&node-id=2688%3A137088&mode=design&t=JMnhoneDpxxvmQfg-1">HDS Design Kit overview</ExternalLink>
 
-_When changing the colour or other styles of component symbols, be sure your custom design adheres to the design guidelines._ Styles within components are carefully considered. We do not recommend detaching the symbol to change default styles.
 
-<Image src="/images/getting-started/designers/component-overrides.png" alt="Customising symbol overrides" style={{ width: '100%', maxwidth: 850, marginbottom: 24 }} viewable/>
+### Customising components
```
Comment (NikoHelle) [inline site/src/docs/getting-started/designer.mdx:63]: Not changed in this PR, but level of this heading and next one ("Styling text elements") seems to be too low? 

Are they meant to be under "Get familiar with HDS"?

Comment (nooranen) [reply site/src/docs/getting-started/designer.mdx:63]: Yes, I suppose, as they refer directly to stuff you can do with the HDS design offering

**Review by NikoHelle — APPROVED** [review]

**Pull request merged**